ibis/backends/duckdb/compiler.py

```diff
@@ -1,3 +1,8 @@
+import duckdb_engine
+from sqlalchemy.ext.compiler import compiles
+
+import ibis.backends.base.sql.alchemy.datatypes as dt
+import ibis.expr.datatypes as ibis_dt
```

Comment:
```suggestion
import ibis.backends.base.sql.alchemy.datatypes as sat
import ibis.expr.datatypes as dt
```

I believe this is the pattern we use when both libraries' type systems are in the mix.